packages/l/libbpf/xmake.lua

```diff
@@ -1,31 +1,38 @@
 package("libbpf")
-
     set_homepage("https://github.com/libbpf/libbpf")
     set_description("Automated upstream mirror for libbpf stand-alone build.")
+    set_license("BSD-2-Clause")
 
     add_urls("https://github.com/libbpf/libbpf/archive/refs/tags/$(version).tar.gz",
              "https://github.com/libbpf/libbpf.git")
+    add_versions("v1.6.2", "16f31349c70764cba8e0fad3725cc9f52f6cf952554326aa0229daaa21ef4fbd")
     add_versions("v0.3", "c168d84a75b541f753ceb49015d9eb886e3fb5cca87cdd9aabce7e10ad3a1efc")
 
-    add_deps("libelf", "zlib")
+    add_deps("zlib")
+
+    on_load(function (package)
+        if package:version() and package:version():lt("0.5") then
+            package:add("deps", "libelf")
+        else
+            package:add("deps", "elfutils")
+        end
+    end)
 
     add_includedirs("include", "include/uapi")
 
-    on_install("linux", "android", function (package)
```

Comment:
it will break xmake/bpf tests and https://github.com/libbpf/libbpf-bootstrap

https://github.com/xmake-io/xmake/actions/runs/17227312910/job/48874434222?pr=6324git

Reply:
https://github.com/libbpf/libbpf-bootstrap/blob/52e97e567b7136cca1fb271be8bf5c4936947eff/examples/c/xmake.lua#L16-L17